Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -11,6 +11,28 @@ $ErrorActionPreference = "Stop"
 [xml]$configfile = Get-Content ".\XML\Other\ignorable-test-warnings.xml"
 $IgnorableWarnings = @($configfile.messages.warnings.keywords.Trim())
 
+# Minor version like 4.3.3.1 doesnt require source code change so the modinfo version will be same as the major version like 4.3.3
+# Hence we are comparing to check whether it is a minor LIS version upgrade
+Function Check-MinorLisVersionUpgrade($LISTarballUrlOld,$LISTarballUrlCurrent) {
+    $oldversion=(($LISTarballUrlOld -split ".*rpms?-")[1] -split ".tar.gz" -split "-")[0]
+    $oldversion=$oldversion.Split(".")
+    Write-LogInfo "LISTarballUrlOld version - $oldversion"
+    $currentversion=(($LISTarballUrlCurrent -split ".*rpms?-")[1] -split ".tar.gz" -split "-")[0]
+    $currentversion=$currentversion.Split(".")
+    Write-LogInfo "LISTarballUrlCurrent version - $currentversion"
+    $minorupgrade=$true
+    for ($i=0; $i -lt 3; $i++) {
```

Comment:
please use descriptive variable names, what is $i?

Reply:
$i is index of an array , so renamed $i to $index